conu/backend/docker/image.py

```diff
@@ -238,19 +238,29 @@ def _run_container(self, run_command_instance, callback):
             container_id = fd.read()
         return container_id, response
 
-    def run_via_binary(self, run_command_instance=None, *args, **kwargs):
+    def run_via_binary(self, command=None, additional_opts=None, *args, **kwargs):
```

Comment:
These changes should also be applied in run_via_binary_in_foreground, so UX would be similar.

Reply:
I agree. But maybe it will be better to do it in follow-up PR, what do you think?

Reply:
If it is possible to do it in this PR it would be great, but I have no problem with merging and creating follow-up issue :)